api_client/python/timesketch_api_client/client.py

```diff
@@ -573,7 +573,7 @@ def refresh_oauth_token(self):
         self.credentials.credential.refresh(request)
 
     def list_sigma_rules(self, as_pandas=False):
-        """Get a list of sigma objects.
+        """DEPRECATED: Get a list of sigma objects.
```

Comment:
Can you add a comment that this is being replaced by the new functions.

Reply:
done